Libraries/Lists/VirtualizedList.js

```diff
@@ -1104,7 +1104,9 @@ class VirtualizedList extends React.PureComponent<Props, State> {
       keyEventHandler = this.props.enableSelectionOnKeyPress
         ? this._handleKeyDown
         : null;
-    } // ]TODO(macOS ISS#2323203)
```

Comment:
If everything in between the "TODO(macOS" comments is mac-only, I actually found the inclusion brackets pretty helpful during merges "[" and "]"

Reply:
Yep.  I moved this one down to line 1109 but forgot the ']' on the new end line.

Reply:
Note in some cases like inside JSX there can be headaches with ES Lint mangling the comments.  That's why in some of the JSX cases I give up and put a single comment at the end.